source/connect/connection-options.txt

```diff
@@ -0,0 +1,286 @@
+.. _cpp-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by using
+options in the connection URI.
```

Comment:
S: since you can use the connection URI or a mongocxx::options::client instance:
```suggestion
available in the {+driver-short+}. You can configure your connection by specifying
options in the connection URI or in an instance of the ``mongocxx::options::client`` class.
```